### PR TITLE
656 update log in

### PR DESCRIPTION
**Pull request by fadeomar**
#### Issues this PR relates to
#656
---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/components/Pages/SignInPage/index.js [hunk 1]
```diff
@@ -114,60 +111,73 @@ export default class SignInPage extends Component {
...
-      </Wrapper>
+            <p style={{ marginTop: 10, marginLeft: 5 }}>
+              <S.ForgetLink to={RESET_PASSWORD}>Forgot password?</S.ForgetLink>
+            </p>
+            {msg && <S.ErrorMsg>{msg}</S.ErrorMsg>}
+            <Button type="secondary" onClick={onFormSubmit}>
+              Sign in
+            </Button>
+            <T.P style={{ textAlign: 'center' }}>
+              dont have an account{' '}
```
Comment (thejoefriel) [inline client/src/components/Pages/SignInPage/index.js:175]: - [ ] Should be 'Don't have an account? Sign up'
- [ ] Sign up should be bold 
- [ ] Color of the first bit should be gray3
- [ ] Need more vertical spacing between this line and the button above. Refer to the wireframes

File: client/src/components/Pages/SignInPage/index.js [hunk 2]
```diff
@@ -114,60 +111,73 @@ export default class SignInPage extends Component {
...
-              pathname: '/',
-              hash: 'findMoreSection',
-            }}
-          >
-            &nbsp;Sign Up
-          </Link>
-        </p>
-      </Wrapper>
+            <p style={{ marginTop: 10, marginLeft: 5 }}>
+              <S.ForgetLink to={RESET_PASSWORD}>Forgot password?</S.ForgetLink>
```
Comment (thejoefriel) [inline client/src/components/Pages/SignInPage/index.js:168]: Please refer to the wireframes for this:
- [ ] Should be underlined
- [ ] Should be 14px not 18px
- [ ] Should be bold 

Also you need to increase padding-left slightly to bring it in line with the Email Password labels etc

File: client/src/components/Pages/SignInPage/index.js [hunk 3]
```diff
@@ -114,60 +111,73 @@ export default class SignInPage extends Component {
...
+            </p>
+            {msg && <S.ErrorMsg>{msg}</S.ErrorMsg>}
+            <Button type="secondary" onClick={onFormSubmit}>
+              Sign in
```
Comment (thejoefriel) [inline client/src/components/Pages/SignInPage/index.js:172]: Should say Log In not Sign In

Comment (fadeomar) [reply client/src/components/Pages/SignInPage/index.js]: done

**Pull request merged**